.evergreen/functions.yml

```diff
@@ -83,6 +83,12 @@ variables:
 
 # This is here with the variables because anchors aren't supported across includes
 post:
+  - command: shell.exec
+    params:
+      working_dir: src
+      shell: bash
+      script: |
+        tar czvf all-e2e-logs.tgz packages/compass-e2e-tests/.log
```

Comment:
This is just incredibly helpful when you want to check whether some log messages occur in all/some of the logs/in which logs exactly, rather than having to download dozens of files individually